packages/runner/src/iframe/iframe.scss

```diff
@@ -94,12 +94,4 @@
     transform-origin: 50% 0;
     width: 100%;
   }
-}
-
-.spec-bridge-iframe {
```

Comment:
these styles were still being injected, but it doesn't really make sense for the style to live here any longer, but inside the `app`, package.